redash-visualizations/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "redash-visualizations",
```

Comment:
Let's name it `@redash/viz`. I've added you to the `redash` org on npm, so you should be able to eventually push it there.